indices.recovery.max_bytes_per_sec may be per-node (#54633)
elastic/elasticsearch: docs/reference/modules/indices/recovery.asciidoc

==== Peer recovery settings

> `indices.recovery.max_bytes_per_sec` (<<cluster-update-settings,Dynamic>>)::

Reviewer: Github won't let me suggest, but we may want to move the parenthetical inside the definition as with `indices.recovery.max_concurrent_file_chunks`.

Limits total inbound and outbound recovery traffic for each node. Defaults to
`40mb`.
+
This limit applies to each node separately. If multiple nodes in a cluster
perform recoveries at the same time, the cluster's total recovery traffic may
exceed this limit.
+
If this limit is too high, ongoing recoveries may consume an excess of bandwidth
and other resources, which can destabilize the cluster.
+

> This is a dynamic setting which means you can set it in each node's
> `elasticsearch.yml` config file and you can update it dynamically using the
> <<cluster-update-settings,cluster update settings API>>. If you set it

Reviewer: I would just add a comma here.
```suggestion
This is a dynamic setting, which means you can set it in each node's
`elasticsearch.yml` config file and you can update it dynamically using the
<<cluster-update-settings,cluster update settings API>>. If you set it
```

dynamically then the same limit applies on every node in the cluster. If you do
not set it dynamically then you can set a different limit on each node, which
is useful if some of your nodes have better bandwidth than others. For example,

> if you are using <<overview-index-lifecycle-management,Index Lifecycle
> Management>> then you may restrict the recovery bandwidth on your warm nodes
> more than on your hot nodes.

Reviewer: Slight reword
```suggestion
if you are use <<overview-index-lifecycle-management,Index Lifecycle
Management>>, you may give hot nodes a higher recovery bandwidth limit than
warm nodes.
```

[float]
==== Expert peer recovery settings
You can use the following _expert_ setting to manage resources for peer
recoveries.

> `indices.recovery.max_concurrent_file_chunks`
> (<<cluster-update-settings,Dynamic>>, Expert):: Number of file chunk requests
> sent in parallel for each recovery. Defaults to `2`.

Reviewer: This doesn't render properly because `::` falls on a different line.
I think we can move the parenthetical info inside the definition like this:

```suggestion
`indices.recovery.max_concurrent_file_chunks`:: 
(<<cluster-update-settings,Dynamic>>, Expert)
Number of file chunk requests sent in parallel for each recovery. Defaults to
`2`.
```
